packages/forms-web-app/src/controllers/projects/documents.js

```diff
@@ -46,6 +47,8 @@ function renderData(req, res, params, response) {
 }
 
 exports.getAboutTheApplication = async (req, res) => {
+  const dt = moment('2019-04-22', 'YYYY-MM-DD').format('YYYY-MM-DD');
```

Comment:
Assuming this and the line below need to be removed and was just used for some debugging?